.github/workflows/release.yml

```diff
@@ -69,7 +69,20 @@ jobs:
       - name: Install packages required for publishing script
         run: |
           sudo apt-get -y update
-          sudo apt-get install -y procmail createrepo awscli reprepro
+          sudo apt-get install -y procmail awscli reprepro
+
+      - name: Build createrepo_c tool
+        run: |
+          git clone https://github.com/rpm-software-management/createrepo_c.git && cd createrepo_c/
```

Comment:
Maybe checkout a tag?

Reply:
Yes, I think need to switch to 0.17.1, as is done in tarantool repo.